### DISCUSSION TITLE
Bad Request for WYZE API

### DISCUSSION THREAD

**Discussion by RodgerDJr** [mrlt8/docker-wyze-bridge#674] [Q&A]
Good morning all .  

I was able to setup the Docker Wyze Bridge and had it working.  Then I did a decided I want to start over to make sure I could do it again.  I guess that was my mistake.  I now cannot get it to work.  

This is the error I am getting in my log.

> 2023/01/16 14:53:30 [WyzeBridge] ☁️ Fetching 'auth' from the Wyze API...
> 2023/01/16 15:11:07 [WyzeBridge] 400 Client Error: Bad Request for url: https://auth-prod.api.wyze.com/user/login
> 2023/01/16 15:12:22 [WyzeBridge] ☁️ Fetching 'auth' from the Wyze API...
> 2023/01/16 15:12:22 [WyzeBridge] 🚷 Invalid credentials?

I have verified that I am passing the correct credentials.  

Here is what my Container looks like.  It is running, but I am not able to view my cameras like I was before.  I was using an older build last time I was using v1.10.0, and now I am using v1.11.1 

![image](https://user-images.githubusercontent.com/49752632/212712146-01a8fc94-80c8-4fcd-b745-97d0c8e30c9c.png)

Any help would be greatly appreciated.  What did I do wrong.  Or is there more I need to do after I have the Container Running.?

Thank you,
Rodger

**Comment by saman79** [comment]
I have the same issue. I noticed that if i try to log in to wyze.com, I get "Error: This account is locked due to failed login attempts. Please try again later."

So I guess this is part of the issue.

Also, I think there might be a limitation with the MFA where only one bridge MFA can be established at a time and since you (and I) did it once and it worked, it didn't work the second time around.

**Comment by ColtonYYZ** [comment]
I'm having the same problem!  It says missing credentials in the log tab but in the configuration tab, they are inputted and they are correct!  It wasn't happening prior to v1.11.2 release.
